Update concepts-run-command.md (#80627)
MicrosoftDocs/azure-docs: articles/azure-vmware/concepts-run-command.md

# Run command in Azure VMware Solution (Preview)

> In Azure VMware Solution, vCenter access is provided with a "CloudAdmin" role. The CloudAdmin role has [certain privileges](concepts-identity.md#view-the-vcenter-privileges) to perform various operations on your vCenter. The Run Command feature enables you to perform certain operations that normally require elevated privileges, via a collection of Powershell Commandlets.

Reviewer: ```suggestion
In Azure VMware Solution, vCenter has a built-in local user called *cloudadmin* assigned to the CloudAdmin role. The CloudAdmin role has vCenter [privileges](concepts-identity.md#view-the-vcenter-privileges) that differ from other VMware cloud solutions and on-premises deployments. The Run command (Preview) feature lets you perform operations that would normally require elevated privileges through a collection of PowerShell cmdlets.
```

Azure VMware Solution supports the following operations:

- [Install and uninstall JetStream DR solution](deploy-disaster-recovery-using-jetstream.md)

- [Configure an external identity source](configure-identity-source-vcenter.md)

- [View and set storage policies](configure-storage-policy.md) 

>[!NOTE]
>Run commands are executed one at a time in the order submitted.

## View the status of an execution

> You can view the status of any previously-executed run command, including the cmdlets' output, errors, warnings, and information logs.

Reviewer: ```suggestion
You can view the status of any executed run command, including the output, errors, warnings, and information logs of the cmdlets.
```

1. Sign in to the [Azure portal](https://portal.azure.com).

1. Select **Run command** > **Run execution status**.

>    You can sort by the various columns by clicking on the column. 

Reviewer: ```suggestion
   You can sort by the various columns by selecting the column. 
```

   :::image type="content" source="media/run-command/run-execution-status.png" alt-text="Screenshot showing Run execution status tab." lightbox="media/run-command/run-execution-status.png":::

> 1. Select the execution you want to view. A pane will appear to the right with details about the execution, as well as other tabs for the various types of output generated by the cmdlet.

Reviewer: ```suggestion
1. Select the execution you want to view. A pane opens with details about the execution, as well as other tabs for the various types of output generated by the cmdlet.
```

   :::image type="content" source="media/run-command/run-execution-status-example.png" alt-text="Screenshot showing an example of a run execution.":::

   You can view more details about the execution including the output, errors, warnings, and information.

>    - **Details** - Summary of the execution details, such as the name, status, package, commandlet name, and terminating error if the command failed. 

Reviewer: ```suggestion
   - **Details** - Summary of the execution details, such as the name, status, package, cmdlet name, and terminating error if the command failed. 
```

   - **Output** - Messages output by the cmdlet. May include progress or the result of the operation. Not all cmdlets have output.

      :::image type="content" source="media/run-command/run-execution-status-example-output.png" alt-text="Screenshot showing the output of a run execution.":::

   - **Error** - Error messages generated in the execution of the cmdlet. This is in addition to the terminating error message on the details pane.

      :::image type="content" source="media/run-command/run-execution-status-example-error.png" alt-text="Screenshot showing the errors detected during the execution of an execution.":::

   - **Warning** - Warnings generated during the execution. 

      :::image type="content" source="media/run-command/run-execution-status-example-warning.png" alt-text="Screenshot showing the warnings detected during the execution of an execution.":::

>    - **Information** - Progress/Diagnostic messages during the execution of a cmdlet.

Reviewer: ```suggestion
   - **Information** - Progress and diagnostic messages during the execution of a cmdlet.
```

Now that you've learned about the Run command concepts, you can use the Run command feature to:

- [Configure storage policy](configure-storage-policy.md) - Each VM deployed to a vSAN datastore is assigned a vSAN storage policy. You can assign a vSAN storage policy in an initial deployment of a VM or when you perform other VM operations, such as cloning or migrating.

> - [Configure external identity source for vCenter](configure-identity-source-vcenter.md) - With the Run command feature, you can configure Active Directory over LDAP or LDAPS for vCenter. This will enable the use of an external identity source as an ActiveDirectory. Then, groups from the external identity source can be added to the CloudAdmin role.

Reviewer: ```suggestion
- [Configure external identity source for vCenter (Run command)](configure-identity-source-vcenter.md) - Configure Active Directory over LDAP or LDAPS for vCenter, which enables the use of an external identity source as an Active Directory. Then, you can add groups from the external identity source to the CloudAdmin role.
```
